components/common/images/Logo.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+import logo from './top_bar_logo.webp';
+
+const Logo = () => {
+  const router = useRouter();
+  function handleClickToHome() {
+    router.push('/');
+  }
+  return (
+    <Image
+      src={logo}
+      onClick={handleClickToHome}
+      style={{ cursor: 'pointer' }}
+      alt="Logo"
+      width={50}
+      height={50}
+    />
+  );
+};
+
+export default Logo;
```

Comment:
Let's not use default exports for components. It's harder to refactor code with default exports and can lead to mistakes.

```
export const Logo = () => {
...
```

and then

```
import { Logo } from 'components/common/images/Logo';
```